### DISCUSSION TITLE
mlforecast making predictions on train data

### DISCUSSION THREAD

**Discussion by Ajinkya-7895** [Nixtla/mlforecast#105] [Q&A]
Hi guys,
Can you suggest some steps by which I can make predictions on train data in mlforecast.
horizon gives the option to predict next steps but I did not come across anything that provides information about using start and end like we have in ARIMA.
I need the model to also make predictions on the train data

**Comment by jmoralez** [comment]
Hey. I think [this answer](https://github.com/Nixtla/mlforecast/issues/106#issuecomment-1478893458) can help, please let us know if this works for you.
